src/index.js

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import ReactDOM from 'react-dom/client';
 import App from './App';
 
-ReactDOM.render(
+ReactDOM.createRoot(document.getElementById('root')).render(
```

Comment:
Great job figuring out how to get rid of the warning in the console! :1st_place_medal:

Reply:
Thank you! I had to search React docs to solve that!